src/plugins/enzymes/src/FindEnzymesDialog.cpp

```diff
@@ -619,6 +618,9 @@ FindEnzymesDialog::FindEnzymesDialog(ADVSequenceObjectContext* advSequenceContex
     connect(cbMaxLength, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &FindEnzymesDialog::sl_maxLengthChanged);
     connect(enzSel, &EnzymesSelectorWidget::si_newEnzimeFileLoaded, this, &FindEnzymesDialog::sl_updateEnzymesVisibilityWidgets);
     connect(cbOverhangType, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &FindEnzymesDialog::sl_updateVisibleEnzymes);
+    connect(cbShowPalindromic, &QCheckBox::toggled, this, &FindEnzymesDialog::sl_updateVisibleEnzymes);
+    connect(cbShowUninterrupted, &QCheckBox::toggled, this, &FindEnzymesDialog::sl_updateVisibleEnzymes);
+    connect(cbShowNondegenerate, &QCheckBox::toggled, this, &FindEnzymesDialog::sl_updateVisibleEnzymes);
```

Comment:
`cbShowNondegenerate` automatically includes `cbShowUninterrupted` results.

What do you think if we make `cbShowNondegenerate` enabled & checked only if 'cbShowUninterrupted' is checked?

Reply:
No, my bad, I checked SnapGene again and found out, that "Nondegenerate" enzymes do not contain external DNA alphabet, but could contain Ns. I will rework it.